client/src/components/Filter.jsx

```diff
@@ -100,11 +107,37 @@ const Filter = ({ filter, setFilter }) => {
                 {sector.toUpperCase()}
               </option>
             ))}
-          </select>
+          </FilterSelect>
         </>
       )}
-    </>
+    </FilterWrapper>
   );
 };
 
 export default Filter;
+
+const FilterWrapper = styled.div`
```

Comment:
I think `form` would be more semantic. :)

Reply:
Done. :-)